examples/distributedLock/README.md

```diff
@@ -0,0 +1,88 @@
+# Example - Distributed Lock APIs
```

Comment:
Shall we create a separate issue for adding this to SDK docs? Or should we update the same in this PR?

Reply:
I would keep this in the same PR.

Reply:
This is example, but we still need SDK doc, right?